lib/asset_checker.rb

```diff
@@ -0,0 +1,39 @@
+class AssetChecker
+  def self.run(argv)
+    assets_file = 'app/assets/javascripts/assets.js.erb'
+    translations_file = 'app/assets/javascripts/i18n-strings.js.erb'
+    @asset_strings = load_included_strings(assets_file)
+    @translation_strings = load_included_strings(translations_file)
+    argv.map { |f| check_file(f) }
+  end
+
+  def self.check_file(file)
+    data = File.open(file).read
+    missing_translations = find_missing(data, /\Wt\(['"](.*)['"]\)/, @translation_strings)
```

Comment:
Probably worth making this capture lazier:

```suggestion
    missing_translations = find_missing(data, /\Wt\(['"](.*?)['"]\)/, @translation_strings)
```

Before: https://regex101.com/r/jW0NiC/1
After: https://regex101.com/r/jW0NiC/2

See: https://www.rexegg.com/regex-quantifiers.html#lazy_solution

Reply:
I see your point with that error, but that also leaves the issue of only matching one translation per line. For completeness, how about something like `(?:\Wt\s?\(['"]([^'^"]*)['"]\))`

https://regexr.com/58kjk

Reply:
> leaves the issue of only matching one translation per line

Would it? Based on the implementation below using `scan`, it seems like it should match multiple within the same line.

```
$ irb
irb(main):001:0> "const x = t('one') + t('two');".scan /\Wt\(['"](.*?)['"]\)/
=> [["one"], ["two"]]
```

Reply:
Not sure - I was basing it off of the regex101 link you left. 

Regardless, the new match group of "anything that's not a quote", is probably more robust anyway. And if we get to the point where there are escaped quotes in the names of translations, there are probably bigger problems in the codebase haha